src/platform/bitboxbase/leds.h

```diff
@@ -12,6 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#ifndef __BITBOXBASE_LEDS_H
+#define __BITBOXBASE_LEDS_H
+
+#include <stdbool.h>
 typedef enum { SMALL_LED, BIG_LED } led_t;
 
+typedef enum { LED_WHITE, LED_RED, LED_GREEN, LED_BLUE, LED_NONE } led_color_t;
```

Comment:
I think it would be nice to prefix the enumerations with `LEDS` (filename). Maybe `LEDS_COLOR_WHITE` or just `LEDS_WHITE`?

Reply:
`LED_COLOR_WHITE`? As the data type is `led_color_t` (Not prefixing the filename, but this is not a function...)